### ISSUE TITLE
cmaps in mpl_interactions image_segmenter

### ISSUE THREAD

**Issue by stefanopiovesan** [ianhi/mpl-interactions#135]
Hi, changing `cmaps `in `mpl_interactions image_segmenter `looks like not working.
I am trying to display grey level images, but I see always `viridis` color any value I set.

```
from mpl_interactions import image_segmenter
from PIL import Image
from skimage.color import rgb2gray

image = Image.open(fname).convert("L")
arr = np.asarray(image)
img = rgb2gray(arr); # img has (x,Y) shape
plt.figure()
#display grey level image
plt.imshow(img, cmap='gray', vmin=0, vmax=255) 

# does not show grey level image
segmenter = image_segmenter(img, mask_colors="red", mask_alpha=0.76, figsize=(7, 7), cmap="gray")
display(segmenter)
```

**Comment by ianhi** [comment]
Hi @stefanopiovesan good catch. That's because apparently the cmap argument doesn't get passed to imshow :grimacing:. The relevant line is here:
https://github.com/ianhi/mpl-interactions/blob/050c989557aecbcb1077306a7b325159ea3c8ecd/mpl_interactions/generic.py#L453

The fix for this is to pass the cmap kwarg along to imshow. Would you like to open a PR?

**Comment by ianhi** [comment]
Also, an easy workaround is to just set the cmap manually after creation like so:

```python
segmenter.displayed.set_cmap("gray")
```

**Comment by ianhi** [comment]
Another nice improvement would be to pass the `vmin` and `vmax` parameters through to imshow. Or more generally accept `**kwargs` and then extract the relevant ones for imshow and use those.

**Comment by stefanopiovesan** [comment]
Hi, this workaround works fine for now. Thanks!

`segmenter.displayed.set_cmap("gray")`

**Comment by ianhi** [comment]
Fixed in version `0.11.0` and up which should be on pypi now (`pip install --upgrade mpl-interactions`). I also fixed a few other things along the way and added some documentation on styling here: https://mpl-interactions.readthedocs.io/en/latest/examples/image-segmentation.html#Styling
